src/main/java/org/terracotta/statistics/ZeroOperationStatistic.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * All content copyright Terracotta, Inc., unless otherwise indicated.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics;
+
+import org.terracotta.statistics.observer.ChainedOperationObserver;
+
+import java.util.Set;
+
+/**
+ * An operation statistic that always return 0 for everything. Used as a null object.
+ */
+public class ZeroOperationStatistic<T extends Enum<T>> implements OperationStatistic<T> {
```

Comment:
@henri-tremblay : this class has been moved from Ehcache

Reply:
Yes. I did it :-) But just noticed something was missing. Now that it's here, we might want to fix it.